buildNativeWin64.bat

```diff
@@ -34,7 +34,7 @@ IF NOT DEFINED GSKIT_64_HOME (
 
 cd src/main/native
 
-@call nmake -nologo -f jgskit.win64.mak clean
-@call nmake -nologo -f jgskit.win64.mak 
+@call nmake -nologo -f jgskit.win64.cygwin.mak clean
```

Comment:
So, this change won't affect the OpenJCEPlusFIPS bundle with Semeru, because the bundle codes call "jgskit.win64.mak" directly without using "buildNativeWin64.bat", right?

Reply:
Exactly. That's why I created a new file to reference here and left the original one intact.